cpp/test/test.cpp

```diff
@@ -143,10 +143,60 @@ void group_list(void)
     DeviceContext d = unwrap(DeviceContext::fromJsonString(deviceContextString));
     IronOxide sdk = unwrap(IronOxide::initialize(d, IronOxideConfig()));
     auto group_list_result = unwrap(sdk.groupList());
-    //We don't create new users in these tests, so all we can do is assert that there is some.
+    // We don't create new users in these tests, so all we can do is assert that there is some.
     TEST_CHECK_(group_list_result.getResult().as_slice().size() > 1, "Group list failed.");
 }
 
+//{
+// "sub": "abcABC012_.$#|@/:;=+'-d1226d1b-4c39-49da-933c-642e23ac1945",
+// "pid": 438,
+// "sid": "ironoxide-dev1",
+// "kid": 593,
+// "iat": 1591901740,
+// "exp": 1591901860
+// }
+void jwt_test_no_prefixes(void)
+{
+    Jwt j = unwrap(Jwt::validate("eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJzdWIiOiJhYmNBQkMwMTJfLiQjfEAvOjs9KyctZDEyMjZkMWItNGMzOS00OWRhLTkzM2MtNjQyZTIzYWMxOTQ1IiwicGlkIjo0MzgsInNpZCI6Imlyb25veGlkZS1kZXYxIiwia2lkIjo1OTMsImlhdCI6MTU5MTkwMTc0MCwiZXhwIjoxNTkxOTAxODYwfQ.wgs_tnh89SlKnIkoQHdlC0REjkxTl1P8qtDSQwWTFKwo8KQKXUQdpp4BfwqUqLcxA0BW6_XfVRlqMX5zcvCc6w"));
+    TEST_CHECK_(j.getAlgorithm().to_std_string() == "ES256", "Wrong jwt algorithm");
```

Comment:
Is there a reason you didn't just `j.getClaims()` and use that repeatedly for the assertions?

Reply:
Nope, I went ahead and changed that here and in the java tests